src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/Utils.cs

```diff
@@ -41,5 +42,10 @@ internal static Uri AppendQueryString(Uri url, string qs)
             builder.Query = newQueryString;
             return builder.Uri;
         }
+
+        internal static bool IsRunningInBrowser()
+        {
+            return RuntimeInformation.IsOSPlatform(OSPlatform.Create("WEBASSEMBLY"));
```

Comment:
Is this what we agreed on with the BCL team?

Reply:
The guidance I got from @pranavkm and @anurse was that this is the pattern going forward

Reply:
This is what we have now. I believe discussions are ongoing and we can react later if necessary. Personally I’d prefer a string like “browser” since WASM is not browser-specific, but 🤷‍♀️

Reply:
The string needs to be browser. Can we circle back and make sure? Last I checked we were leaning in that direction. I worry about future breaking changes

Reply:
@pranavkm @danroth27 @SteveSandersonMS What's the story here? Is there a final decision on the string to use for detecting the browser? We'd really like to get this in and right the first time before 3.1.4 closes on Tuesday.

Reply:
It's changing to `BROWSER` See: https://github.com/mono/mono/pull/19482. Also an issue for this: https://github.com/dotnet/runtime/issues/33328#issuecomment-596072263

Reply:
FYI @lewing

Reply:
Is there any idea when that'll happen? I guess we'll need to either wait, or change the string at the same time they change it

Reply:
The change should land before 3.2-RC. Since this change aligns with the RTM release, using `BROWSER` might be the correct thing to do. Again @lewing in case you think otherwise.

Reply:
We can change it now to `BROWSER`. The timing should be fine (3.1.4 is shipping no earlier than 3.2 GA, by the current schedule). It's not going to *fail* if the string is wrong, right? It just won't light up the fixed behavior until you're running on a runtime with the updated OS platform string.

Reply:
> It's not going to fail if the string is wrong, right? It just won't light up the fixed behavior until you're running on a runtime with the updated OS platform string.

That'd be my understanding, it won't fail, it just won't do anything until the corresponding change is made. I'll make the change